protodroid/src/main/java/id/lukasdylan/grpc/protodroid/internal/database/ProtodroidDatabase.kt

```diff
@@ -5,7 +5,7 @@ import androidx.room.Database
 import androidx.room.Room
 import androidx.room.RoomDatabase
 
-@Database(entities = [ProtodroidDataEntity::class], version = 1, exportSchema = false)
+@Database(entities = [ProtodroidDataEntity::class], version = 2, exportSchema = false)
```

Comment:
You need to provide Migration as this library is already used on some projects

Reply:
Oh yes, I will add an AutoMigration? Will clean it up when I clean up the merge conflicts

Reply:
Oh AutoMigration is in 2.4.0-alpha01, maybe just a manual at this stage